client/components/upgrade-nudge/index.jsx

```diff
@@ -0,0 +1,82 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import classNames from 'classnames';
+import noop from 'lodash/noop';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import Gridicon from 'components/gridicon';
+import analytics from 'analytics';
+import sitesList from 'lib/sites-list';
+
+const sites = sitesList();
+
+export default React.createClass( {
+
+	displayName: 'UpgradeNudge',
+
+	propTypes: {
+		onClick: React.PropTypes.func,
+		className: React.PropTypes.string,
+		message: React.PropTypes.string,
+		icon: React.PropTypes.string,
+		event: React.PropTypes.string,
+		href: React.PropTypes.string,
+		jetpack: React.PropTypes.bool
+	},
+
+	getDefaultProps() {
+		return {
+			onClick: noop,
+			message: 'And get your own domain address.',
+			icon: 'star',
+			event: null,
+			jetpack: false
+		}
+	},
+
+	handleClick() {
+		if ( this.props.event ) {
+			analytics.tracks.recordEvent( 'calypso_upgrade_nudge_cta_click', {
+				cta_name: this.props.event
+			} );
+		}
+		this.props.onClick();
+	},
+
+	render() {
+		const classes = classNames( this.props.className, 'upgrade-nudge' );
+		const site = sites.getSelectedSite();
```

Comment:
Could we reduxify it?
We have selectedSite in redux tree I think, and everything should be wrapped in a provider already...